sys/dev/isp/ispreg.h

```diff
@@ -266,6 +266,25 @@ typedef struct {
 #define ISP28XX_BASE_ADDR	0x7f7d0000
 #define ISP28XX_FLT_ADDR	(0x11000 / 4)
 
+#if 0
```

Comment:
Why is this zeroed out?

Reply:
This block was already added for my upcoming rewrite of the NVRAM and FLT parts.
As it does not belong to this pull request, I've removed it for now.